### ISSUE TITLE
Delay before running background queries

### ISSUE THREAD

**Issue by ulriknyman** [DEIS-Tools/H-Uppaal#38]
The queries that are executed in the background to check the reachability to locations should have a wait time added.

When creating a new location this location will have no edges leading to it, thus it will be clear that it will generate a warning when there is no path to the location. We should have a wait period X such that the check will only happen after this amount of time has passed after a location has been created. 

We can later tweak how long this wait time should be.

**Comment by Nielswps** [comment]
Should the delay be extended when the location is being edited? Like Emil proposed at the meeting

**Comment by ulriknyman** [comment]
I think the optimal solution is one where the query is executed when the location has been left alone for X amount of time. 

So a timer is reset whenever the location is edited. When this timer reaches X, the query is run. 

I do not know if this is easy to implement, and I am open to other suggestions.

**Comment by Nielswps** [comment]
I can see that the background queries for reachability has been implemented such that they run as soon as the model has been left alone. I have added a delay to this, hence having them wait X amount of time before running the queries after the model has been sitting unchanged.

Furthermore, I have added a Boolean property on the location that is checked before the "No incoming edges" warning is shown. This way, whenever the location is being altered, the property can be set to avoid the warning popping up.
